README.md

```diff
@@ -18,9 +18,72 @@ Or install it yourself as:
 
     $ gem install grpc-kit
 
-## Usage
+## Documentation
 
-TODO: Write usage instructions here
+- [`GRPC::Kit::Communication::Resilient`](#grpckitcommunicationresilient)
+- [`GRPC::Kit::Logger`](#grpckitlogger)
+- [`GRPC::Kit::Queue::Publisher`](#grpckitqueuepublisher)
+- [`GRPC::Kit::Queue::Worker`](#grpckitqueueworker)
+
+### GRPC::Kit::Communication::Resilient
+
+Sometimes, GRPC and Google::Cloud raises _unavailable error_, but you can retry and all works fine. Based on [Datastore documentation](https://cloud.google.com/appengine/articles/handling_datastore_errors#timeouts-due-to-datastore-issues) we implemented a _helper_ to _resilient communication_.
+
+Example:
+
+```ruby
+pubsub = Google::Cloud::Pubsub.new
+topic  = pubsub.create_topic('topic_name')
+topic.publish('message') # receives Google::Cloud::UnavailableError
+
+# But you can uses resilient communication
+include GRPC::Kit::Communication::Resilient
+topic = resilient { topic.publish('message') }
+
+# And you can sets the limit of repetition...
+topic = resilient(limit: 5) { topic.publish('message') }
+```
+
+### GRPC::Kit::Logger
+
+By default all logs sent to `GRPC.logger` are ignored. But `GRPC::Kit::Logger` configure automatically logs to use _STDOUT_.
+
+### GRPC::Kit::Queue::Publisher
+
+To publish a message to topic (creating if not exists) you need only configure environment variables for `Google::Cloud::Pubsub` and use:
```

Comment:
- Ordem das palavras: "you **only** need **to** configure.."

- Em inglês não é permitido sujeito oculto ou inexistente como em português. Pra preencher buracos você poder usar `it` ou `one`: "creating `one` if `none` exists"